packages/block-library/src/site-logo/style.scss

```diff
@@ -2,4 +2,11 @@
 	.aligncenter {
 		display: table;
 	}
+
+	// Variations
+	&.is-style-rounded img {
+		// We use an absolute pixel to prevent the oval shape that a value of 50% would give
+		// to rectangular images. A pill-shape is better than otherwise.
+		border-radius: 9999px;
+	}
```

Comment:
@jasmussen while adding this, I noticed that it's technically not necessary because the original `is-style-rounded` image block rule is not scoped to just the image block:

https://github.com/WordPress/gutenberg/blob/c043d0f6cd737caddf626925097034903fce5974/packages/block-library/src/image/style.scss#L63-L68

I think I should probably go back into the image block style there and make it specific to `wp-block-image`... I don't think that'll break anything, right?

Reply:
Makes sense to me

Reply:
Good catch.

> To keep each block's code self-contained, I think I should probably go back into the image block style there and make it specific to wp-block-image... I don't think that'll break anything, right?

Does it add any specificity?

Reply:
Right now, there is none — the image block's rule just targets `.is-style-rounded img`. I'd recommend updating to `.wp-block-image.is-style-rounded img` instead. I doubt anyone uses this rule outside of the image block (and if they do, they probably shouldn't be doing so).

Reply:
👍 👍 🚢

Reply:
Great, I've opened a separate PR to handle that: https://github.com/WordPress/gutenberg/pull/27649